src/util/report.py

```diff
@@ -105,6 +105,9 @@ def add_to_migration_reports(message, category="Other", format="text"):
         else:
             raw = True
     migration_reports.setdefault(category, []).append((message, raw))
+    migration_reports_length = sum(len(msg) for reps in migration_reports.values() for msg, _ in reps)
```

Comment:
We could count the category title as well. Just in case somebody passes a huge chunk of text :/
```suggestion
    migration_reports_length = sum(len(msg) for reps in migration_reports.values() for msg, _ in reps) + sum(map(len, migration_reports))
```